### PR TITLE
Sync Modes Release Blog Post

### PR DESCRIPTION
**Pull request by pedroslopez**
To be published on release of sync modes

### CODE REVIEW

File: pages/blog/using-sync-modes.mdx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+image: "sync-modes/sync-modes-breakdown.png"
+tags: [engineering, product]
+---
+
+A few weeks ago we [wrote about Sync Modes](/blog/sync-modes) and why they may be useful when it comes to syncing data to a destination. In short, Sync Modes allow you to have more control over what operations are performed and how Grouparoo interacts with contacts that may already exist in the destination system.
+
+In Grouparoo v0.3, we're introducing Sync Modes as a core concept and making them available throughout all destinations. In this initial release, we're rolling out sync modes for [Salesforce](/integrations/destinations/salesforce), [Intercom](/integrations/destinations/intercom), [Hubspot](/integrations/destinations/hubspot), [Mailchimp](/integrations/destinations/mailchimp), [Pipedrive](/integrations/destinations/pipedrive), [Marketo](/integrations/destinations/marketo), [Facebook](/integrations/destinations/facebook), and [Zendesk](/integrations/destinations/zendesk). We'll continue adding to this list by updating remaining plugins and build them into new destinations going forward.
```
Comment (andyjih) [inline pages/blog/using-sync-modes.mdx]: SEO, SEO, SEO 😆

**Review by andyjih — APPROVED** [review]
👍

**Comment by evantahler** [comment]
Sync modes are released! @pedroslopez merge when ready!

**Comment by evantahler** [comment]
You may want to update the date

**Pull request merged**